doc/source/examples/sharp-immersed-boundary/sedimentation-1-particle/sedimentation-1-particle.rst

```diff
@@ -81,8 +81,8 @@ Physical Properties
 * The ``kinematic viscosity`` is set to  0.6041666666666. This value is derived from the case description by dividing :math:`\mu_f` by :math:`\rho_f`.
 
 .. note:: text
-	The fluid density is not set here as the sharp interface solver has his own definition of the density of the fluid. This will be modified soon to unify both definitions in the parameter. 
-	
+    The fluid density is not set here as the sharp interface solver has its own definition of the density of the fluid. This will be modified soon to unify both definitions in the parameter.
```

Comment:
Is this still an issue? if so, clearly you should put in the project of things to fix

Reply:
It is no longer an issue. Sorry for letting this one pass. Nice catch.